app/models/banana.rb

```diff
@@ -1,6 +1,6 @@
 class Banana < ActiveRecord::Base
   def cut
-    "cut cut!"
+    'cut cut!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.